src/backend/dx12/src/window.rs

```diff
@@ -3,23 +3,40 @@ use core::{self, image};
 use dxguid;
 use std::collections::VecDeque;
 use std::ptr;
+#[cfg(feature = "winit")]
 use winit;
```

Comment:
nit: could do `use winit::Window;` right away

Reply:
Should we make other backends (previous pull requests) consistent with it?

Reply:
sure thing, although this is really minor